sdks/python/apache_beam/transforms/stats.py

```diff
@@ -214,7 +212,7 @@ def create_accumulator(self, *args, **kwargs):
 
```

Comment:
Suggestion (Maybe add a JIRA (does not have to be fixed in this PR) for future):
- Pass a hash_fn argument to ApproximateUniqueCombineFn, that can be passed by users to customize the hash_fn they would like to use. It can default to `hash`

Reply:
Ticket is created: https://issues.apache.org/jira/browse/BEAM-7525